packages/@coorpacademy-components/src/atom/chip/index.js

```diff
@@ -0,0 +1,90 @@
+import React, {useMemo, useState, useCallback} from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import get from 'lodash/fp/get';
+import Color from 'colorjs.io';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import Provider from '../provider';
+import {COLORS} from '../../variables/colors';
+import style from './style.css';
+
+const LUMINOSITY_DELTA = 0.08;
+const {cm_primary_blue: DEFAULT_BACKGROUND_COLOR} = COLORS;
+const ICON_SIZE = '12px';
+
+export const calculateLuminosity = (l, luminosityDelta = LUMINOSITY_DELTA) =>
```

Comment:
on sait mal compris, je voulais dire tout le color to tostring. (d'ailleurs t'as oublié de le brancher)
```suggestion
export const calculateLuminosity = (color, luminosityDelta = LUMINOSITY_DELTA) => new Color(color)
    .to('hsl')
    .set({l: l => l * (1 - luminosityDelta)})
    .toString();
```